content-src/asrouter/docs/targeting-attributes.md

```diff
@@ -485,3 +486,13 @@ Boolean pref that gets set the first time the user opens the FxA toolbar panel
 ```ts
 declare const hasAccessedFxAPanel: boolean;
 ```
+
+### `hasWhatsNewPanelEnabled`
```

Comment:
`hasWhatsNewPanelEnabled` looks funny. Should it be `isWhatsNewPanelEnabled` similar to `isFxAEnabled`?

Reply:
I say this now and realize we already reviewed that patch... This PR needs to be rebased I guess

Reply:
Renamed